discovery-provider/src/api/v1/models/challenges.py

```diff
@@ -8,3 +8,14 @@
         "attestation": fields.String(required=True),
     },
 )
+
+undisbursed_challenge = ns.model(
+    "undisbursed_challenge",
+    {
+        "challenge_id": fields.String(required=True),
+        "user_id": fields.String(required=True),
+        "specifier": fields.String(),
```

Comment:
for an undisbursed challenge this will always be there. it was nullable in the get_challenges resp bc we roll up aggregates